### PR TITLE
Add instructions for install dino via homebrew

### PR DESCRIPTION
**Pull request by marat-yusupov**
Add instructions for install dino via homebrew and resolve possible errors

### CODE REVIEW

File: BUILD_MACOS.md [hunk 1]
```diff
@@ -20,7 +20,21 @@ cd dino
...
+
+<code>$ brew tap marat-yusupov/homebrew-dino</code>
+
+<code>$ brew install marat-yusupov/homebrew-dino/dino</code>
+
+You can start the Dino client installed via homebrew with the following command:
+
+<code>$ ./opt/homebrew/Cellar/dino/3/bin/dino</code>
+
+I recommend creating a shortcut with a symbolic link to this file so that you can open it in a way that is convenient for you.
```
[the remark this answers is not in the document]
Comment (marat-yusupov) [reply BUILD_MACOS.md:33]: +

File: BUILD_MACOS.md [hunk 2]
```diff
@@ -20,7 +20,21 @@ cd dino
 
 ## Install with Brew
 
-TODO
+For using this formula, follow next instructions:
+
+<code>$ brew tap marat-yusupov/homebrew-dino</code>
```
[the remark this answers is not in the document]
Comment (mxlgv) [reply BUILD_MACOS.md:25]: @marat-yusupov  Сreated: https://github.com/mxlgv/homebrew-dino

File: BUILD_MACOS.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mxlgv) [reply BUILD_MACOS.md]: @marat-yusupov Why were html tags used in the markdown file?

[the remark this answers is not in the document]
Comment (marat-yusupov) [reply BUILD_MACOS.md]: @mxlgv As far as I know, HTML tags can be used in markdown files (I checked), but I replaced them with quotes to make it consistent

**Pull request merged**